framework/web_app/src/boot/i18n.ts

```diff
@@ -10,24 +10,24 @@ export type MessageSchema = typeof messages['en-US'];
 // See https://vue-i18n.intlify.dev/guide/advanced/typescript.html#global-resource-schema-type-definition
 /* eslint-disable @typescript-eslint/no-empty-interface */
```

Comment:
i have not looked into what this file does yet, looks possibly useful (i think it's for character/language support in web) we can either address now because the file is being edited or make a todo

this is up for grabs if you want

Reply:
Did some research:

https://quasar.dev/options/app-internationalization/

Don't see the benefit to configuring this file unless we are planning to support any other languages.

Seems like you can also use it for some datetime formatting OR by default datetimes use the browsers settings, which also seems fine. We can do some testing to see if we care, e.g., a User in Australia has incorrect browser settings.

Reply:
interesting, will leave this for now in case we want to in the very end (unlikely but if the app shoots off in popularity...maybe?)